src/adapters/default/attendance.adapter.ts

```diff
@@ -144,8 +144,15 @@ export class AttendanceService {
     });
     let studentArray = [];
     for (let value of arrayIds) {
-      const response = await axios.get(`${this.secondUrl}/${value}`, request);
-      const data = response.data;
+      let config = {
+        method: "get",
+        url: `${this.secondUrl}/${value}`,
```

Comment:
Variable names - please use a name that signifies what the variable is meant for.

In this case it may be `studentAPIUrl`

Reply:
@coolbung sir comment solved